README.md

```diff
@@ -3,64 +3,91 @@
 ## Learning Goals
 
 - Define web programming
-- Identify the “Three Pillars of Web Programming"
+- Identify reference example
+- Identify the "Three Pillars of Web Programming"
 
 ## Introduction
 
-HTML, CSS and JavaScript are the main tools we use to build the web. To imagine
-how they all work together, think about the web as a robot. HTML is the robot's
-structure, CSS is the paint on the robot, plus how its pieces are positioned,
-and JavaScript is what actually makes the robot move. So if we want to build web
-applications that accept, change and return information (in other words, make
-the robot _move_), we'll need understand the Document Object Model (DOM), the
-JavaScript language and the principles of web programming.
+"Front-End Web Programming" is a phrase that gets used in a lot of places by a
+lot of people. In this lesson we're going to create a common definition of "web
+programming" and show one _tiny_ example that demonstrates all of "web
+programming's" parts.
 
 ### Define Web Programming
 
-Web programming, in the most general sense, is how we markup and code web sites
-and web applications. Usually when we talk specifically about web programming,
-we are focusing on the part where we are working with data in some way: taking
-it in, pushing it out, saving, changing or deleting it. For this, we need a
-programming language. JavaScript is the language designed to work on the front-
-end, or user-facing part, of the web.
+Web programming, at its heart, is:
 
-While this might be your first introduction to web programming as a programmer,
-you've probably experienced plenty of web interactions that someone else
-programmed for you. For example, have you ever "favorited" a post on social
-media? Chances are that interaction went something like this:
+> Creating documents with HTML, styling / positioning the documents' content
+> with CSS and updating that content and servers based on events using
+> JavaScript.
```

Comment:
@jenmyers do you agree with this definition? once we lock this in we should put it in your "bible" for this project so that everyone else uses the same metaphor.

Reply:
clearly that can be written better